lib/commands/context.js

```diff
@@ -117,12 +113,21 @@ commands.mobileGetContexts = async function mobileGetContexts () {
   return await webviewHelpers.getWebViewsMapping(this.adb, opts);
 };
 
-helpers.switchContext = async function switchContext (name) {
+helpers.assignContexts = function assignContexts (webviewsMapping) {
+  const webviews = this.isChromeSession ?
+    [CHROMIUM_WIN] :
+    webviewHelpers.parseWebviewNames(webviewsMapping, {ensureWebviewsHavePages: this.opts.ensureWebviewsHavePages});
```

Comment:
this string is a bit too long, could we split it to make the line more readable?

Reply:
Addressed as https://github.com/appium/appium-android-driver/pull/699/commits/8818f5bfc1ef2f23a9b711a857a0dc484785df95